### PR TITLE
Use responders

### PR DESCRIPTION
**Pull request by jordanmaguire**
Use responders!

### CODE REVIEW

**Comment by lordmortis** [comment]
¡Seems good!

File: config/locales/responders.en.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    actions:
+      create:
+        notice: '%{resource_name} was successfully created.'
+        # alert: '%{resource_name} could not be created.'
+      update:
+        notice: '%{resource_name} was successfully updated.'
+        # alert: '%{resource_name} could not be updated.'
+      destroy:
+        notice: '%{resource_name} was successfully destroyed.'
+        alert: '%{resource_name} could not be destroyed.'
```
Comment (osiro) [inline config/locales/responders.en.yml]: Can have destroyed replaced with "removed" or "deleted"?

**Comment by osiro** [comment]
:+1:

File: config/locales/responders.en.yml (hunk 1 above)
Comment (jordanmaguire) [inline config/locales/responders.en.yml]: good idea :+1:

**Pull request merged**